spec/models/barcode_item_spec.rb

```diff
@@ -58,17 +58,29 @@
       it "->barcodeable_id shows only barcodes for a specific barcodeable_id" do
         global_barcode_item
         create(:global_barcode_item, base_item: create(:base_item))
-        results = BarcodeItem.barcodeable_id(base_item.id)
+        items = BarcodeItem.barcodeable_id(base_item.id)
```

Comment:
This felt more descriptive, and matches what we used in our new/altered tests below.